src/mlpack/methods/dbscan/dbscan_main.cpp

```diff
@@ -86,26 +88,48 @@ void RunDBSCAN(RangeSearchType rs = RangeSearchType())
 
   // Load dataset.
   arma::mat dataset = std::move(CLI::GetParam<arma::mat>("input"));
-
   const double epsilon = CLI::GetParam<double>("epsilon");
   const size_t minSize = (size_t) CLI::GetParam<int>("min_size");
-
-  DBSCAN<RangeSearchType> d(epsilon, minSize, !CLI::HasParam("single_mode"),
-      rs);
-
-  // If possible, avoid the overhead of calculating centroids.
   arma::Row<size_t> assignments;
-  if (CLI::HasParam("centroids"))
-  {
-    arma::mat centroids;
-
-    d.Cluster(dataset, assignments, centroids);
 
-    CLI::GetParam<arma::mat>("centroids") = std::move(centroids);
+  // Check if random selection is used.
+  if (CLI::HasParam("random_selection")) // Random Selection
+  {
+    DBSCAN<RangeSearchType, RandomPointSelection> d(epsilon, minSize, 
+        !CLI::HasParam("single_mode"), rs);
+
+    // If possible, avoid the overhead of calculating centroids.
+    if (CLI::HasParam("centroids"))
+    {
+      arma::mat centroids;
+
+      d.Cluster(dataset, assignments, centroids);
+
+      CLI::GetParam<arma::mat>("centroids") = std::move(centroids);
+    }
+    else
+    {
+      d.Cluster(dataset, assignments);
+    }
   }
-  else
+  else // Ordered Selection
```

Comment:
Please remove this comment from here. Also, if possible try to accomodate the comments on the line above the code, to be consistent with rest of the codebase.